### PR TITLE
Add rst ref in supported op documentation

### PR DESCRIPTION
**Pull request by Kh4L**
Signed-off-by: Serge Panev <spanev@nvidia.com>

### CODE REVIEW

**Comment by JanuszL** [comment]
Builds 610251

File: dali/python/nvidia/dali/ops.py [hunk 1]
```diff
@@ -43,6 +43,7 @@ def _docstring_generator(cls):
 
     schema = b.GetSchema(op_name)
     ret = pre_doc
+    ret += '.. _' + op_name + ':\n'
```
Comment (klecki) [inline dali/python/nvidia/dali/ops.py]: Operators already have reference links, as you can get a direct link, for example: https://docs.nvidia.com/deeplearning/sdk/dali-developer-guide/docs/supported_ops.html#nvidia.dali.ops.BBoxPaste
No idea how to get it in python though.

File: docs/supported_op_devices.py [hunk 2]
```diff
@@ -13,7 +13,7 @@ def main(argv):
     all_ops = cpu_ops.union(gpu_ops).union(mix_ops).union(support_ops)
     op_name_max_len = len(max(all_ops, key=len))
     name_bar = op_name_max_len * '='
-    formater = '{:{c}>{op_name_max_len}}  {:{c}>6}  {:{c}>6}  {:{c}>6}  {:{c}>7}\n'
+    formater = ':ref:`{0:{c}>{op_name_max_len}}<{0}>`  {1:{c}>6}  {2:{c}>6}  {3:{c}>6}  {4:{c}>7}\n'
```
[the remark this answers is not in the document]
Comment (klecki) [reply docs/supported_op_devices.py]: I am not sure if this won't be an overkill, but maybe we should link back from operator to this table as well?

**Comment by JanuszL** [comment]
What I managed to do in https://github.com/JanuszL/DALI/tree/ref_in_docs_able is:
image
@Kh4L won't work as modification in `_docstring_generator` doesn't do anything  (or at least I don't see any impact), nor we cannot reference existing tags as they are uperrcase and rst converts them to lowercase.

**Comment by JanuszL** [comment]
OK. Finally figured that out - https://github.com/NVIDIA/DALI/pull/519.

**Pull request closed without merging**